src/Microsoft.Crank.Controller/JobSerializer.cs

```diff
@@ -55,7 +55,7 @@ public static async Task InitializeDatabaseAsync(
         {
             var createCmd =
                 @"
-                IF OBJECT_ID(N'dbo." + tableName + @"', N'U') IS NULL
+                IF NOT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND TABLE_NAME = @TableName)
                 BEGIN
                     CREATE TABLE [dbo].[" + tableName + @"](
```

Comment:
This one is not parameterized, so the issue is still there. But this can't be parameterized, so you can updated your changes with something like this:

```c#
var encodedTableName = new SqlCommandBuilder().QuoteIdentifier(unquotedIdentifier: tableName);
```